services/app-api/forms/mcpar.json

```
@@ -4397,7 +4389,7 @@
                 "validation": "number",
                 "props": {
                   "label": "D1.XIII.3 Total standard and expedited PA requests",
-                  "hint": "Enter the total number of standard and expedited PA requests logged by the plan during the prior calendar year. If you choose not to respond prior to June 2026, enter \"NR\" for not reporting."
+                  "hint": "Enter the total number of standard and expedited PA requests logged by the plan during the prior calendar year. If you choose not to respond prior to June 2026, enter “NR“ for not reporting."
```

fix double open quote
```suggestion
                  "hint": "Enter the total number of standard and expedited PA requests logged by the plan during the prior calendar year. If you choose not to respond prior to June 2026, enter “NR” for not reporting."
```